src/test/components/Details.test.js

```diff
@@ -14,42 +15,62 @@ import { getProfileFromTextSamples } from '../fixtures/profiles/processed-profil
 import { tabSlugs } from '../../app-logic/tabs-handling';
 import type { TabSlug } from '../../app-logic/tabs-handling';
 
+// Let's mock all possible views so that we don't spend too much time rendering stuff.
+// We use the tab slugs as class names. `call-tree` is an exception because if
+// we need a dash to masquerade as custom elements so that React doesn't emit a
+// warning.
+jest.mock('../../components/calltree/ProfileCallTreeView', () => 'call-tree');
+jest.mock('../../components/flame-graph', () => 'flame-graph');
+jest.mock('../../components/stack-chart', () => 'stack-chart');
+jest.mock('../../components/marker-chart', () => 'marker-chart');
+jest.mock('../../components/marker-table', () => 'marker-table');
+jest.mock('../../components/network-chart', () => 'network-chart');
+jest.mock('../../components/js-tracer', () => 'js-tracer');
```

Comment:
I'm not really happy about that.
Note I can't use a `forEach` loop because there's a check that we don't use any non-constants (except some whitelisted things), and Jest+Babel refuses to run.

Reply:
Potential follow-up: Is there any way we can ensure that we don't over-render here when we add new components to the details view? Since we don't use shallow rendering, it would be easy to over-render.

Reply:
yeah, I don't have a good idea :(
A comment in the tabSlugs definition?